tests/testthat/test-workflow-bbi.R

```diff
@@ -177,5 +177,39 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
     })
   })
 
+
+  test_that("wait_for_nonmem() correctly reads in stop time [BBR-UTL-012]", {
+    # create model
+    mod1 <- read_model(file.path(MODEL_DIR_BBI, "1"))
+    submit_model(mod1, .mode = "local", .wait = FALSE)
+    wait_for_nonmem(list(mod1), 100, interval = 5)
+    expect_true(suppressMessages(!rlang::is_empty(read_table(file.path(MODEL_DIR_BBI, 1, "1.tab"), skip = 1))))
+  })
+
+  test_that("wait_for_nonmem() doesn't error out if no stop time found [BBR-UTL-013]", {
```

Comment:
Let's simplify this test because we don't want to use SGE in our test suite, in part because it makes things like CI more of a pain. Plus it's not really within `bbr`'s purview to test whether SGE is hooked up right (only that we pass the right thing to it, and we test that in `bbi`).

Try this instead (after you've added the warning I mention above)

```r
test_that("wait_for_nonmem() runs to time_limit if no stop time found [BBR-UTL-013]", {
    # model setup

    mod_fail <- copy_model_from(
      read_model(file.path(MODEL_DIR_BBI, "1")),
      "failure"
    )
    .p <- submit_model(mod_fail, .mode = "local", .wait = FALSE)
    .p$process$kill()

    #wait_for_nonmem(mod_fail, 3, interval = 1) # hopefully this can go away now?

    expect_warning(
      wait_for_nonmem(mod_fail, 2, interval = 1), # dont need high wait time since we know it failed
      "[whatever 'your model didnt finish' warning we're going to add]"
    )
  })
```